src/core_plugins/kibana/common/tutorials/functionbeat_instructions.js

```diff
@@ -0,0 +1,468 @@
+      commands: [
+        'curl -L -O https://artifacts.elastic.co/downloads/beats/functionbeat/functionbeat-{config.kibana.version}-darwin-x86_64.tar.gz',
+        'tar xzvf functionbeat-{config.kibana.version}-darwin-x86_64.tar.gz',
+        'cd functionbeat-{config.kibana.version}-darwin-x86_64/',
+      ],
+    },
+    LINUX: {
+      title: i18n.translate('kbn.common.tutorials.functionbeatInstructions.install.linuxTitle', {
+        defaultMessage: 'Download and install Functionbeat',
+      }),
+      textPre: i18n.translate('kbn.common.tutorials.functionbeatInstructions.install.linuxTextPre', {
+        defaultMessage: 'First time using Functionbeat? See the [Getting Started Guide]({link}).',
+        values: { link: '{config.docs.beats.functionbeat}/functionbeat-getting-started.html' },
+      }),
+      commands: [
+        'curl -L -O https://artifacts.elastic.co/downloads/beats/functionbeat/functionbeat-{config.kibana.version}-linux-x86_64.tar.gz',
+        'tar xzvf functionbeat-{config.kibana.version}-linux-x86_64.tar.gz',
+        'cd functionbeat-{config.kibana.version}-linux-x86_64/',
+      ],
+    },
+    WINDOWS: {
+      title: i18n.translate('kbn.common.tutorials.functionbeatInstructions.install.windowsTitle', {
+        defaultMessage: 'Download and install Functionbeat',
+      }),
+      textPre: i18n.translate('kbn.common.tutorials.functionbeatInstructions.install.windowsTextPre', {
+        defaultMessage: 'First time using Functionbeat? See the [Getting Started Guide]({functionbeatLink}).\n\
+ 1. Download the Functionbeat Windows zip file from the [Download]({elasticLink}) page.\n\
+ 2. Extract the contents of the zip file into {folderPath}.\n\
+ 3. Rename the {directoryName} directory to `Functionbeat`.\n\
+ 4. Open a PowerShell prompt as an Administrator (right-click the PowerShell icon and select \
+**Run As Administrator**). If you are running Windows XP, you might need to download and install PowerShell.\n\
+ 5. From the PowerShell prompt, go to the Functionbeat directory:',
+        values: {
+          directoryName: '`functionbeat-{config.kibana.version}-windows`',
+          folderPath: '`C:\\Program Files`',
+          functionbeatLink: '{config.docs.beats.functionbeat}/functionbeat-getting-started.html',
+          elasticLink: 'https://www.elastic.co/downloads/beats/functionbeat',
+        },
+      }),
+      commands: [
+        'PS > cd C:\\Program Files\\Functionbeat',
+      ],
+    }
+  },
+  DEPLOY: {
+    OSX_LINUX: {
+      title: i18n.translate('kbn.common.tutorials.functionbeatInstructions.deploy.osxTitle', {
+        defaultMessage: 'Deploy Functionbeat to AWS Lambda',
+      }),
+      textPre: i18n.translate('kbn.common.tutorials.functionbeatInstructions.deploy.osxTextPre', {
+        defaultMessage: 'This installs Functionbeat as a Lambda function.\
+The `setup` command checks the Elasticsearch configuration and loads the \
+Kibana index pattern. It is normaly safe to omit this command.',
```

Comment:
"normally" is mistyped

Reply:
also, this would be an ES index pattern instead of Kibana right?

Reply:
Setup loads a Kibana index pattern. The Elasticsearch template is here part of the `Elasticsearch configuration`.